WcaOnRails/config/initializers/translations.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+if defined?(Rails::Server) # See: https://stackoverflow.com/a/44441168
+  Rails.configuration.after_initialize do
+    modification_timestamp = Timestamp.find_or_create_by!(name: 'en_translation_modification')
+    latest_modification_date = DateTime.parse(`git log -1 --format='%ai' #{Rails.root.to_s}/config/locales/en.yml`)
+    if modification_timestamp.date != latest_modification_date
```

Comment:
1. Hmm if we deploy something involving the change of `en.yml` translators gets notified; then we roll it back and they get notified again; when we fix stuff and merge again, yet another notification. Maybe it's not that bad, but at least super confusing.
On the other hand the original version they will have in the Internationalize app is the one taken from GitHub, so whatever we do on the production, it doesn't impact the version they should translate. Does this seem better to you?